packages/block-library/src/media-text/edit.js

```diff
@@ -229,81 +234,123 @@ function MediaTextEdit( { attributes, isSelected, setAttributes } ) {
 	};
 
 	const mediaTextGeneralSettings = (
-		<PanelBody title={ __( 'Settings' ) }>
-			<RangeControl
-				__nextHasNoMarginBottom
-				__next40pxDefaultSize
-				label={ __( 'Media width' ) }
-				value={ temporaryMediaWidth || mediaWidth }
-				onChange={ commitWidthChange }
-				min={ WIDTH_CONSTRAINT_PERCENTAGE }
-				max={ 100 - WIDTH_CONSTRAINT_PERCENTAGE }
-			/>
-			<ToggleControl
-				__nextHasNoMarginBottom
+		<ToolsPanel
+			label={ __( 'Settings' ) }
+			resetAll={ resetAll }
+			dropdownMenuProps={ TOOLSPANEL_DROPDOWNMENU_PROPS }
+		>
+			{ mediaUrl && (
```

Comment:
I think it would be better to remove this conditional statement. Because always rendering this control was intentional in #58447. See [this comment](https://github.com/WordPress/gutenberg/pull/58447#pullrequestreview-1851744275).